### PR TITLE
Replace ad hocery with science

### PR DESCRIPTION
**Pull request by matklad**
bors r+
🤖

### CODE REVIEW

File: crates/ra_project_model/src/sysroot.rs [hunk 1]
```diff
@@ -126,7 +126,6 @@ core
 alloc
 collections
 libc
-panic_unwind
```
Comment (bjorn3) [inline crates/ra_project_model/src/sysroot.rs:129]: Why was this removed?

Comment (matklad) [reply crates/ra_project_model/src/sysroot.rs:129]: There's already one bellow (hence the dedup call in fst construction code, just to be safe :D)
